Rubberduck.VBEEditor/ComManagement/TypeLibs/TypeLibWrapper.cs

```diff
@@ -64,6 +64,7 @@ private TypeLibTextFields CachedTextFields
                     var cache = new TypeLibTextFields();
                     // as a C# caller, it's easier to work with ComTypes.ITypeLib
                     ((ComTypes.ITypeLib)_target_ITypeLib).GetDocumentation((int)KnownDispatchMemberIDs.MEMBERID_NIL, out cache._name, out cache._docString, out cache._helpContext, out cache._helpFile);
+                    if (cache._name == null) cache._name = "[VBA.Immediate.Window]";
```

Comment:
Could a non VBA `ITypeLib` pass through here? If so, if it happened to have null for the name, woudln't it get mislabeled?